pkg/segment/writer/stats/segstats_test.go

```diff
@@ -62,6 +62,31 @@ func Test_addSegStatsNums(t *testing.T) {
 
 }
 
+func Test_addSegStatsNumsMixed(t *testing.T) {
```

Comment:
You added basically the same test in packer_test.go. Is it useful to have this test in both places?

Reply:
These are 2 similar functions that are used one is on the packer/writer side and the other is one the reader side. Notice [AddSegStatsNums](https://github.com/siglens/siglens/blob/49fe395f2be32cb1a3321c136492afa4ab6f53ed/pkg/segment/writer/stats/segstats.go#L42) and [addSegStatsNums](https://github.com/siglens/siglens/blob/49fe395f2be32cb1a3321c136492afa4ab6f53ed/pkg/segment/writer/packer.go#L1581). I think we have copies because of ingestion performance where we do not want checks like hasEvalFunc etc. which are present in the AddSegStatsNums but not in addSegStatsNums. But not sure on this.